DfE.FindInformationAcademiesTrusts/Pages/Schools/Ofsted/SafeguardingAndConcerns.cshtml

```diff
@@ -0,0 +1,85 @@
+﻿@page
+@using System.ComponentModel
+@using DfE.FindInformationAcademiesTrusts.Data.Enums
+@using DfE.FindInformationAcademiesTrusts.Extensions
+@model SafeguardingAndConcernsModel
+
+@{
+    Layout = "_SchoolLayout";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h3 class="govuk-heading-m" data-testid="subpage-header">
+            @Model.PageMetadata.SubPageName
+        </h3>
+
+        <details class="govuk-details">
+            <summary class="govuk-details__summary">
+                <span class="govuk-details__summary-text">
+                    What this information means
+                </span>
+            </summary>
+            <div class="govuk-details__text">
+                <p class="govuk-body">
+                    <span class="govuk-!-font-weight-bold">'Effective safeguarding'</span> means that a school has been judged to have adequate safeguarding policies and processes in place. It reviews them regularly and implements them well.
+                </p>
+                <p class="govuk-body">
+                    <span class="govuk-!-font-weight-bold">'Category of concern'</span> means that a school has been found inadequate in a key or provision judgement. The school either has ‘serious weaknesses’ or requires ‘special measures.’
```

Comment:
Non-blocking: it looks like there's some smart quotes here (highlighted in red below), for consistency I'd suggest using straight quotes instead.

<img width="1185" height="345" alt="" src="https://github.com/user-attachments/assets/b802e21c-a1b7-4510-ae8b-4bc8c1015964" />